### PR TITLE
Add `examples` within the `docs/`

### PR DESCRIPTION
**Pull request by alvarobartt**
## Description

This PR adds the `examples` to the https://hf.co/docs/google-cloud documentation, so as to add more content there.

### CODE REVIEW

File: docs/source/resources.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- [Deploy Meta Llama 3.1 405B on Google Cloud Vertex AI](https://huggingface.co/blog/llama31-on-vertex-ai)
+
+## Documentation
+
+- [Google Cloud Hugging Face Deep Learning Containers](https://cloud.google.com/deep-learning-containers/docs/choosing-container#hugging-face)
+- [Google Cloud public Artifact Registry for DLCs](https://console.cloud.google.com/artifacts/docker/deeplearning-platform-release/us/gcr.io)
+- [Serve Gemma open models using GPUs on GKE with Hugging Face TGI](https://cloud.google.com/kubernetes-engine/docs/tutorials/serve-gemma-gpu-tgi)
+- [Generative AI on Vertex - Use Hugging Face text generation models](https://cloud.google.com/vertex-ai/generative-ai/docs/open-models/use-hugging-face-models)
+
+## Examples
```
Comment (pagezyhf) [inline docs/source/resources.mdx:19]: I think we can remove this part now it's available on the left  side navigation bar?

File: docs/source/resources.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# 📄 Resources, Documentation & Examples
```
Comment (pagezyhf) [inline docs/source/resources.mdx:1]:
```suggestion
# 📄 Other Resources
```

File: docs/source/resources.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Learn how to use Hugging Face in Google Cloud by reading our blog posts, documentation and examples below.
+
+## Blog posts
+
+- [Hugging Face and Google partner for open AI collaboration](https://huggingface.co/blog/gcp-partnership)
+- [Google Cloud TPUs made available to Hugging Face users](https://huggingface.co/blog/tpu-inference-endpoints-spaces)
+- [Making thousands of open LLMs bloom in the Vertex AI Model Garden](https://huggingface.co/blog/google-cloud-model-garden)
+- [Deploy Meta Llama 3.1 405B on Google Cloud Vertex AI](https://huggingface.co/blog/llama31-on-vertex-ai)
+
+## Documentation
```
Comment (pagezyhf) [inline docs/source/resources.mdx:12]:
```suggestion
## Google Documentation
```

File: docs/source/resources.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# 📄 Resources, Documentation & Examples
+
+Learn how to use Hugging Face in Google Cloud by reading our blog posts, documentation and examples below.
```
Comment (pagezyhf) [inline docs/source/resources.mdx:3]:
```suggestion
Learn how to use Hugging Face in Google Cloud by reading our blog posts and Google documentation
```

**Comment by philschmid** [comment]
> that's great! 2 feedbacks:
> 
> * Would be good to define standards in example names for easier discoverability
> * I think we could separate better how to deploy from the Hub, from the Model Garden in GCP and from a notebook. I don't want to mix this in this PR, so i'll make a suggestion once this PR is merged.

I would keep it if automated. As it might be better for SEO. Its always good to have a full overview.

**Comment by alvarobartt** [comment]
Thank you for the review @philschmid and @pagezyhf!

So for the examples and containers listing I want to start moving some scripts I use locally into a separate GitHub Action so that those are automatically generated yes. In this case, for the "additional" example listing I think we can keep it, but I don't know much about best SEO practices either.

**Review by jeffboudier — APPROVED** [review]
LGTM, great improvements! I like the new example navigation structure. I think we can keep the list of examples duplicated in the "more resources" page.

File: docs/source/resources.mdx (hunk 4 above)
Comment (pagezyhf) [reply docs/source/resources.mdx:3]:
```suggestion
Learn how to use Hugging Face in Google Cloud by reading our blog posts, Google documentation and examples below.
```

**Pull request merged**